overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -12057,7 +12057,7 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "This is the additive that will bring our diesel fuel to the next level.\n\nMix §9Nitric Acid§r, §9Ethenone§r, and §9Acetic Acid§r in a §3Chemical Reactor§r to make §9Tetranitromethane§r.",
+          "desc:8": "This is the additive that will bring our diesel fuel to the next level.\n\nMix §9Nitric Acid§r and §9Ethenone§r in a §3Chemical Reactor§r to make §9Tetranitromethane§r.\n\nIn a newly added route, you can also mix §9Nitric Acid§r and §9Methyl Acetate§r in a §3Chemical Reactor§r.",
```

Comment:
I'm not sure it makes sense to refer to it as a newly added route as a new player wouldn't have the context of knowing what we're comparing it to.

I'm also thinking it could make sense to add a quest for Methyl Acetate and let the player choose which route to use. Ethenone is literally only used for TNM, but Methyl Acetate comes from reacting acetic acid and methanol (and a bit from tower-distilling wood vinegar which gives all three of these chemicals) and it can be reprocessed into other useful chemicals as well.

Reply:
It would be a newly added route because it did not exist in the current 1.2.2 versions of the game, as I believe this recipe was something that was added through the GTCE chemistry changes.

As for adding a quest, that could be fine, although that area is a little cramped.